client/app/containers/index.js

```diff
@@ -83,17 +84,31 @@ class App extends React.Component {
     return api.getTree(newPath).then(mapToState).catch(this.handleError);
   }
 
-  componentDidMount() {
-    const newPath = window.location.search || this.state.path;
+  createBreadcrumbs(path) {
+    let breadcrumbs = path.split('/');
 
-    this.formNewState(newPath).then(newState => {
-      window.history.replaceState(newState, null, newPath);
-      window.onpopstate = ({ state }) => this.setState(state);
-    }).catch(this.handleError);
+    breadcrumbs.splice(0, 1, '.');
```

Comment:
Yes, because first item in path is "?path=.'.
So I have two options - remove first item from breadcrumbs and create root link in Breadcrumb.
Or, replace first item with dot. This is little weird, but it's working. ¯\_(ツ)_/¯

I can pick first option.